surveys/2024-annual-survey/translations/fr.md

```diff
@@ -2,29 +2,29 @@
 
 Type: select one
 
-- Oui, j'utilise Rust (pour tout usage, même si vous êtes en phase d'apprentissage).
-- Non, je n'utilise pas Rust actuellement, mais je l'ai fait dans le passé.
-- Non, je n'ai jamais utilisé Rust.
+- Oui, j’utilise Rust (pour tout usage, même si vous êtes en phase d'apprentissage).
+- Non, je n’utilise pas Rust actuellement, mais je l’ai fait dans le passé.
+- Non, je n’ai jamais utilisé Rust.
 
-### Vous avez indiqué que vous n’utilisez plus Rust, pour quelle raison avez-vous arrêté d'utiliser Rust ?
+### Vous avez indiqué que vous n’utilisez plus Rust, pour quelle raison avez-vous arrêté d’utiliser Rust ?
 
 Type: select all that apply
```

Comment:
Should this be translated?

Reply:
this is just some text to visually separate questions and answer options when copying the survey on the survey platform. It will be discarded.

Reply:
Ah seems not.